colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala

```diff
@@ -104,6 +106,8 @@ case class MetricContext(namespace: MetricAddress, collection: Collection) exten
 case class MetricSystem private[metrics] (namespace: MetricAddress, collectionIntervals : Map[FiniteDuration, MetricInterval],
                                           collectionSystemMetrics : Boolean, config : Config) extends MetricNamespace {
 
+  private val localHostname = java.net.InetAddress.getLocalHost.getHostName
+  val extraTags: TagMap = Map("host" -> localHostname)
```

Comment:
This needs to be optional.  In the StatsReporter there's a `includeHostnameInGlobalTags` option.  I'm ok with merging this PR without this change, but an issue needs to be created first.